repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -348,6 +348,14 @@ export interface BasicService {
     serviceDescription: string;
 }
 
+export interface ReturnService extends BasicService {
+    id: number;
+}
+
+export interface ReturnServiceWithErrors {
```

Comment:
We have a generic type called WithErrors which you can use here, for example WithErrors<ReturnService>.

Reply:
using that one